src/explore-education-statistics-admin/src/pages/methodology/edit-methodology/content/components/MethodologyAccordionSection.tsx

```diff
@@ -153,7 +153,7 @@ const MethodologyAccordionSection = ({
         )}
         renderEditableBlock={block => (
           <MethodologyEditableBlock
-            //allowImages
+            allowImages
```

Comment:
There is a second allowImages related to Releases that also needs to be uncommented